packages/@atjson/renderer-plain-text/src/index.ts

```diff
@@ -1,10 +1,21 @@
 import Document, { Annotation } from "@atjson/document";
+import { List } from "@atjson/offset-annotations";
 import Renderer from "@atjson/renderer-hir";
 
 export default class PlainTextRenderer extends Renderer {
   constructor(document: Document, ...args: any[]) {
     document
-      .where((annotation: Annotation) => annotation.type !== "parse-token")
+      .where(
```

Comment:
Does it make sense to break this out to a `supportedAnnotations` array, or perhaps just add a bit of documentation to make it clear that/why we're stripping unsupported annotations here?

Reply:
@tim-evans might have more context here - i'm happy to add comments around this

Reply:
Yeah, I think that'd be a good idea to add a comment that we're explicitly removing annotations we don't support here.